### PR TITLE
Updates to Entity Analytics Dashboard

### LINKED ISSUE #4314: Updates to Entity Analytics Dashboard (missed in 8.11 + inconsistencies)
## Description 

The following Entity Analytics Dashboard updates have been raised by @SourinPaul ([Google doc reference](https://docs.google.com/document/d/1vKOEULMPxGB0piqBbRZTHoSkTCLjEmD14h7woE0rEAc/edit#heading=h.ymbqkdc56e21)): 

> 
> In 8.11, [the dashboard page](https://www.elastic.co/guide/en/security/current/detection-entity-dashboard.html) includes the below section:
> image
> 
> Desired Changes to bullets identified by 1& 2 (see above screenshot) :
> 1. In 8.11, users can enable the new risk engine. The documentation should take users to https://www.elastic.co/guide/en/security/current/advanced-entity-analytics-overview.html.  
>   a. We may move references to the legacy enablement process to the legacy documentation pages ([Host](https://www.elastic.co/guide/en/security/current/verify-host-risk-score.html)).
>   b. A few additional references (screenshosts below) to Risk Scoring in the Dashboard take the user to legacy documentation. We should centralize these links in the dashboard page or ensure they route users correctly.
> 
> image
> image
> 
> Note: In 8.11, Platinum+ users can view a call-out to enable/ upgrade the risk engine on the Entity Analytics Dashboard
> 
> 2. Enablement of Jobs may not be a hard requirement to use the dashboard, but a callout. We must clarify:
>   a. Users must explicitly ‘install and run’ anomaly jobs before viewing outputs from the jobs in the dashboard. They can enable those that are relevant to them.
>   b. The list now only captures 6 of the 41+ jobs. Do we need to capture the list of jobs here? Instead, we can direct users to the [prebuilt jobs page](https://www.elastic.co/guide/en/security/current/prebuilt-ml-jobs.html).
>   c. Clarify that users cannot add custom anomaly jobs from their environment to the EA dashboard as of this release.

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves #4314.

Preview: [Entity Analytics dashboard](https://security-docs_4345.docs-preview.app.elstc.co/guide/en/security/master/detection-entity-dashboard.html)

### CODE REVIEW

File: docs/dashboards/entity-dashboard.asciidoc [hunk 1]
```diff
@@ -74,18 +67,26 @@ Interact with the table to filter data, view more details, and take action:
...
 == Anomalies
 
-Anomalies identify suspicious or irregular behavior patterns. The Anomalies table displays the total number of host and user anomalies identified by six predefined {ml} jobs (named in the Anomaly name column). These jobs must be installed and running to provide anomaly data.   
+Anomaly detection jobs identify suspicious or irregular behavior patterns. The Anomalies table displays the total number of host and user anomalies identified by these prebuilt {ml} jobs (named in the **Anomaly name** column).
+
+.Requirements
+[sidebar]
+-- 
+
+To display anomaly data, you must {ml-docs}/ml-ad-run-jobs.html[install and run] one or more <<prebuilt-ml-jobs, prebuilt anomaly detection jobs>>. Custom anomaly detection jobs cannot be added to the Entity Analytics dashboard.
```
Comment (jmikell821) [inline docs/dashboards/entity-dashboard.asciidoc:82]: Flip to active voice:
```suggestion
To display anomaly data, you must {ml-docs}/ml-ad-run-jobs.html[install and run] one or more <<prebuilt-ml-jobs, prebuilt anomaly detection jobs>>. You cannot add custom anomaly detection jobs to the Entity Analytics dashboard.
```

**Review by jmikell821 — COMMENTED** [review]
Looks great! Just one small comment. I will "approve" to not block the merging, but let's wait for @SourinPaul's approval before merging.

**Review by jmikell821 — DISMISSED** [review]

File: docs/dashboards/entity-dashboard.asciidoc [hunk 2]
```diff
@@ -74,18 +67,26 @@ Interact with the table to filter data, view more details, and take action:
...
 == Anomalies
 
-Anomalies identify suspicious or irregular behavior patterns. The Anomalies table displays the total number of host and user anomalies identified by six predefined {ml} jobs (named in the Anomaly name column). These jobs must be installed and running to provide anomaly data.   
+Anomaly detection jobs identify suspicious or irregular behavior patterns. The Anomalies table displays the total number of host and user anomalies identified by these prebuilt {ml} jobs (named in the **Anomaly name** column).
+
+.Requirements
+[sidebar]
+-- 
+
+To display anomaly data, you must {ml-docs}/ml-ad-run-jobs.html[install and run] one or more <<prebuilt-ml-jobs, prebuilt anomaly detection jobs>>. You cannot add custom anomaly detection jobs to the Entity Analytics dashboard.
```
Comment (SourinPaul) [inline docs/dashboards/entity-dashboard.asciidoc:82]: Does it make sense to simply? 

`The Anomalies table displays the total number of host and user anomalies identified` to `The Anomalies table displays the total number of anomalies identified`

**Review by SourinPaul — CHANGES_REQUESTED** [review]

File: docs/dashboards/entity-dashboard.asciidoc [hunk 3]
```diff
@@ -74,18 +67,26 @@ Interact with the table to filter data, view more details, and take action:
...
+
+-- 
 
 [role="screenshot"]
 image::images/anomalies-table.png[Anomalies table]
 
-If data is missing, click the *Run job* link next to a {ml} job to install and start the job. 
+If anomaly data is missing, click the *Run job* link next to a {ml} job to install and start the job.
```
[the remark this answers is not in the document]
Comment (SourinPaul) [reply docs/dashboards/entity-dashboard.asciidoc:89]: @ajosh0504 @machadoum, do you recall why we added this explicitly?

**Review by jmikell821 — DISMISSED** [review]

File: docs/dashboards/entity-dashboard.asciidoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (SourinPaul) [reply docs/dashboards/entity-dashboard.asciidoc:89]: Thanks, @machadoum, for confirming.

@natasha-moore-elastic I suggest removing line 88 entirely, as it is redundant to line 82. Thanks!

**Review by SourinPaul — APPROVED** [review]

**Review by SourinPaul — APPROVED** [review]

**Review by jmikell821 — APPROVED** [review]

**Pull request merged**